geth-poa/start.sh

```diff
@@ -12,7 +12,7 @@ source ./common_start.sh
 node_start() {
   # geth is dumb and won't let us run it in the background, and nohup redirects to file when run in a script
   nohup geth \
-    --networkid 12346 \
+    --networkid "$(cat "$ROOT/networkid")" \
     --datadir "${ROOT}/chain" \
```

Comment:
so networked is a file with network id we want? so we could have many network ids, dev-node-geth and dev-pop-geth could have their own?